### DISCUSSION TITLE
Why recommend single quote but use double quote in code?

### DISCUSSION THREAD

**Discussion by tiendq** [saleor/saleor#11044] [Q&A] [answered]
In [contribution guide](https://github.com/saleor/saleor/blob/main/CONTRIBUTING.md#editorconfig) you recommend people to follow Black and use [`.editorconfig`](https://github.com/saleor/saleor/blob/main/.editorconfig) which has `quote_type = single` but in code I see double quotes. Why?

**Comment by peelar** [comment, marked as answer]
Hello @tiendq 👋 

Good catch! There seems to be some inconsistency indeed. Right now, please treat Black as the source of truth. We will adjust the `.editorconfig` file accordingly. Thank you 🙇
